pkg/jobrunaggregator/jobtableprimer/job_name_generator.go

```diff
@@ -0,0 +1,263 @@
+		"https://raw.githubusercontent.com/openshift/release/master/core-services/release-controller/_releases/release-ocp-%s-ci.json",
+		"https://raw.githubusercontent.com/openshift/release/master/core-services/release-controller/_releases/release-ocp-%s-multi.json",
+		"https://raw.githubusercontent.com/openshift/release/master/core-services/release-controller/_releases/release-ocp-%s-ppc64le.json",
+		"https://raw.githubusercontent.com/openshift/release/master/core-services/release-controller/_releases/release-ocp-%s-s390x.json",
+		"https://raw.githubusercontent.com/openshift/release/master/core-services/release-controller/_releases/release-ocp-%s.json",
+	}
+)
+
+func newJobNameGenerator() *jobNameGenerator {
+	generator := &jobNameGenerator{
+		periodicURLs: []string{
+			"https://raw.githubusercontent.com/openshift/release/master/ci-operator/jobs/openshift/release/openshift-release-master-periodics.yaml",
+			"https://raw.githubusercontent.com/openshift/release/master/ci-operator/jobs/openshift/multiarch/openshift-multiarch-master-periodics.yaml",
+
+			"https://raw.githubusercontent.com/openshift/release/master/ci-operator/jobs/openshift/release/openshift-release-release-4.10-periodics.yaml",
+			"https://raw.githubusercontent.com/openshift/release/master/ci-operator/jobs/openshift/release/openshift-release-release-4.11-periodics.yaml",
+			"https://raw.githubusercontent.com/openshift/release/master/ci-operator/jobs/openshift/release/openshift-release-release-4.12-periodics.yaml",
+			"https://raw.githubusercontent.com/openshift/release/master/ci-operator/jobs/openshift/release/openshift-release-release-4.13-periodics.yaml",
+			"https://raw.githubusercontent.com/openshift/release/master/ci-operator/jobs/openshift/release/openshift-release-release-4.14-periodics.yaml",
+			"https://raw.githubusercontent.com/openshift/release/master/ci-operator/jobs/openshift/release/openshift-release-release-4.15-periodics.yaml",
+			"https://raw.githubusercontent.com/openshift/release/master/ci-operator/jobs/openshift/release/openshift-release-release-4.16-periodics.yaml",
+
+			"https://raw.githubusercontent.com/openshift/release/master/ci-operator/jobs/openshift/hypershift/openshift-hypershift-release-4.13-periodics.yaml",
+			"https://raw.githubusercontent.com/openshift/release/master/ci-operator/jobs/openshift/hypershift/openshift-hypershift-release-4.14-periodics.yaml",
+			"https://raw.githubusercontent.com/openshift/release/master/ci-operator/jobs/openshift/hypershift/openshift-hypershift-release-4.15-periodics.yaml",
+			"https://raw.githubusercontent.com/openshift/release/master/ci-operator/jobs/openshift/hypershift/openshift-hypershift-release-4.16-periodics.yaml",
+		},
+		releaseConfigURLs: []string{
+			"https://raw.githubusercontent.com/openshift/release/master/core-services/release-controller/_releases/release-ocp-4.10-arm64.json",
```

Comment:
same nit - do we even need this any longer or can we just use the templates for all the defined releases?